Rakefile

```diff
@@ -153,6 +153,13 @@ task "assets:precompile" do
   fail unless $?.success?
 end
 
+desc "Validate, lint, format OpenAPI YAML file"
+task :openapifmt do
```

Comment:
We have `:linter` namespace. We can move it to there similar to other linters

Reply:
`rake linter` runs all of them: `rubocop`, `brakeman`, `erb_formatter`.

Reply:
I'll put it in there.